package.json

```diff
@@ -84,6 +84,7 @@
     "ejs": "^2.5.6",
     "eslint-plugin-eslint-plugin": "^0.7.1",
     "eslint-plugin-node": "^4.2.1",
+    "eslint-plugin-throws-on-load": "^1.0.0",
```

Comment:
I tried doing this:

```diff
diff --git a/tests/lib/config/plugins.js b/tests/lib/config/plugins.js
index 8bbff180..433cbeb8 100644
--- a/tests/lib/config/plugins.js
+++ b/tests/lib/config/plugins.js
@@ -43,7 +43,10 @@ describe("Plugins", () => {
                 "eslint-plugin-example": plugin,
                 "@scope/eslint-plugin-example": scopedPlugin,
                 "./environments": Environments,
-                "../rules": Rules
+                "../rules": Rules,
+                "eslint-plugin-throws-on-load"() {
+                    throw new Error("error thrown while loading this module");
+                }
             });
         });
 

```

But it doesn't work -- it seems like the function that throws an error is getting returned as the module rather than getting called. Is there another way to do it?